com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Utility/HDRP/Water/EvaluateWaterFoamData.cs

```diff
@@ -0,0 +1,79 @@
+using Usage = UnityEditor.ShaderGraph.GraphDelta.GraphType.Usage;
+
+namespace UnityEditor.ShaderGraph.Defs
+    {
+        internal class EvaluateWaterFoamDataNode : IStandardNode
+        {
+            public static string Name => "EvaluateWaterFoamData";
+            public static int Version => 1;
+
+            public static FunctionDescriptor FunctionDescriptor => new(
+                Name,
+    @"FoamData foamData;
+ZERO_INITIALIZE(FoamData, foamData);
+EvaluateFoamData(SurfaceGradientInput, LowFrequencySurfaceGradient, SimulationFoam, customFoam, UV0.xyz, foamData);
```

Comment:
The c is CustomFoam needs to be capitalized to match the one in the ParameterDescriptor.